### PR TITLE
Feature/docker script

### PR DESCRIPTION
**Pull request by wesleymonte**

### CODE REVIEW

File: docker/docker.sh [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  docker run -dit \
+    --name "${ARREBOL_CONTAINER}" \
+    -p ${PORT}:8080 \
+    -v "$(pwd)"/src/main/java/resources/arrebol.json:/service/config/arrebol.json \
+    -v "$(pwd)"/src/main/java/resources/application.properties:/service/config/application.properties \
+    "${ARREBOL_REPO}":"${TAG}"
+}
+
+publish() {
+  local tag="${1:-latest}"
```
Comment (raonismaneoto) [inline docker/docker.sh:30]: ${1:-latest} or ${1-:latest} ?

Comment (raonismaneoto) [reply docker/docker.sh:30]: or even ${1-latest}

**Review by raonismaneoto — CHANGES_REQUESTED** [review]

File: docker/docker.sh (hunk 1 above)
Comment (wesleymonte) [reply docker/docker.sh:30]: This means to use the `latest` value by default if the `$1` variable is empty.

Comment (raonismaneoto) [reply docker/docker.sh:30]: isnt the ':' a typo?

**Review by raonismaneoto — CHANGES_REQUESTED** [review]

File: docker/docker.sh (hunk 1 above)
Comment (wesleymonte) [reply docker/docker.sh:30]: It is not.

**Review by raonismaneoto — APPROVED** [review]

File: docker/docker.sh (hunk 1 above)
Comment (thiagomanel) [reply docker/docker.sh:30]: @wesleymonte add a comment explaining this ":". it is not trivial

Comment (wesleymonte) [reply docker/docker.sh:30]: Done.

**Review by thiagomanel — APPROVED** [review]
lgtm

**Pull request merged**